test/e2e/e2e.go

```diff
@@ -187,9 +64,6 @@ var _ = ginkgo.SynchronizedAfterSuite(func() {
 			e2elog.Logf("Error gathering metrics: %v", err)
 		}
 	}
```

Comment:
Should the `Run only Ginkgo on node 1` part of cleanup also be moved into an utility function?

Reply:
Maybe `gatherTestSuiteMetrics()` should go to an utility function as well.